packages/terra-list/src/List.scss

```diff
@@ -1,16 +1,62 @@
-@import '~terra-mixins';
 @import './variables';
 
-.terra-List {
-  list-style: none;
-  margin: 0;
-  padding: 0;
-}
+// List Component
+// ========================================================
+/* stylelint-disable selector-class-pattern, selector-max-compound-selectors */
+:local {
+  .list {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  .divided > .list-item {
+    border-top: solid 1px $terra-list-divider-color;
```

Comment:
Thanks, Emily. Update it in this [commit](https://github.com/cerner/terra-core/pull/648/commits/2b1c01c7e246b6bb3a35a78af13099a84f348a1a)